__tests__/lint/assertions-casing-camel-case-error/.redocly.yaml

```
@@ -4,16 +4,17 @@ apis:
 
 lint:
   rules:
-    assertions:
-      - context:
-        - type: Operation
-          matchParentKeys: [get]
-        subject: Operation
-        property: operationId
-        message: Operation id for get operation should be canmelCase
-        casing: camelCase
-      - subject: Operation
-        property: operationId
-        message: Operation id should be camelCase
-        casing: camelCase
+    assert/operation-get:
+      context:
+      - type: Operation
+        matchParentKeys: [get]
+      subject: Operation
+      property: operationId
+      message: Operation id for get operation should be canmelCase
```

```suggestion
      message: Operation id for get operation should be camelCase
```

I think this is an opportunity to fix this typo. The corresponding snapshot will need to be updated too.